Loop54.Tests.Shared/CallMethods.cs

```diff
@@ -57,6 +75,24 @@ public void SearchWithCustomData([Values("steak", "chicken breast")]string query
             Assert.AreEqual("pong", responseMessage);
         }
 
+        [Test]
+        public void GetPopularEntitiesHasResults()
+        {
+            var request = new GetPopularEntitiesRequest("click", null, null);
+            var response = IgnoreTestIfEndpointNotFound(c => c.GetPopularEntities(WrapRequest(request)));
```

Comment:
The call to IgnoreTestIfEndpointNotFound can be removed once we know the HelloWorld engine has been upgraded, but at least this way the tests will automatically start running as soon as it is upgraded. (I checked that they pass with a clone of the HelloWorld engine on a branch that has the new APIs.)